website/.remarkrc.mjs

```diff
@@ -1,22 +1,24 @@
-import packageJson from "./package.json" assert { type: "json" };
+// TODO: Enable the plugin once a more performant solution is found
+// import packageJson from "./package.json" assert { type: "json" };
+//
+// export default {
+//   plugins: [
+//     "@react-native-website/remark-lint-no-broken-external-links",
+//     {
+//       skipUrlPatterns: [
+//         // False positive, flagged as a bot and rate limited
+//         "www.apkfiles.com",
+//         // TODO: replace the 2048 example repository with another suitable project
+//         "github.com/JoelMarcey",
+//       ],
+//       baseUrl: "https://reactnative.dev/docs",
+//       headers: {
+//         "user-agent": `${packageJson.name}/${packageJson.version}`,
+//       },
+//     },
+//   ],
+// }
```

Comment:
did you intend to remove this code?

Reply:
@cortinico I just have moved it from the plugins array down in the config (https://github.com/facebook/react-native-website/pull/4537/files#diff-9c907a2040c63723d9ff2c30aa8edf803f4f564a91d4fc6762664444e4b7ede9L5) to the top level, since comment in array was causing parsing issues. It was already commented out on `main`:
* https://github.com/facebook/react-native-website/blob/b0feb2f7f6993ab07789e8c6b1edc35c0600e361/website/.remarkrc.mjs#L4-L21